OMPython/__init__.py

```diff
@@ -62,6 +62,21 @@
 import pyparsing
 from distutils import spawn
 
+# The following import are added by Sudeep
+import numpy as np
+import pandas as pd
+import csv
+
+import matplotlib.pyplot as plt
```

Comment:
Is it wise to depend on matplotlib when it causes the compiler to use exponential memory? https://trac.openmodelica.org/OpenModelica/ticket/3764